check.go

```diff
@@ -51,8 +52,8 @@ type CustomEventCheckFragment struct {
 }
 
 type ServiceOwnershipCheckFragment struct {
-	RequireContactMethod *bool                             `graphql:"requireContactMethod"`
-	ContactMethod        *ServiceOwnershipCheckContactType `graphql:"contactMethod"`
+	RequireContactMethod *bool        `graphql:"requireContactMethod"`
+	ContactMethod        *ContactType `graphql:"contactMethod"`
```

Comment:
I recall this was put in for a reason - something about the field actually supporting more then just the ContactType because the API has it as a string.

Unless there is a reason to be undoing this lets not do this.